docker_images/nemo/app/pipelines/automatic_speech_recognition.py

```diff
@@ -58,7 +58,7 @@ def __call__(self, inputs: np.array) -> Dict[str, str]:
             transcriptions = self.model.transcribe([audio_path])
 
             # if transcriptions form a tuple (from RNNT), extract just "best" hypothesis
-            if type(transcriptions) == tuple and len(transcriptions) == 2:
+            if type(transcriptions) is tuple and len(transcriptions) == 2:
```

Comment:
Modified so that the code quality test pass (needed for `flake8` latest version / `pycodestyle>=`[v2.11.0](https://github.com/PyCQA/pycodestyle/blob/2.11.0/pycodestyle.py#L1452))